python/paddle/v2/framework/tests/gradient_checker.py

```diff
@@ -90,6 +90,9 @@ def product(dim):
     # we only compute gradient of one element each time.
     # we use a for loop to compute the gradient of every element.
     for i in xrange(tensor_size):
+        for var_name in input_values:
```

Comment:
add a parameter to choose if use inplace or not to speed up gradient check

Reply:
Thx. Already added, with in-place option = False as default.